src/app/src/components/FacilityListSummary.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { string } from 'prop-types';
+
+const facilityListSummaryStyles = Object.freeze({
+    nameStyles: Object.freeze({
+        color: '#000 !important',
+        fontWeight: 500,
+        fontSize: '20px',
+        margin: '8px 0',
+    }),
+    descriptionStyles: Object.freeze({
+        color: '#000 !important',
+        marginBottom: '24px',
+    }),
+});
+
+
+function FacilityListSummary({ name, description }) {
+    return (
+        <div>
+            <div style={facilityListSummaryStyles.nameStyles}>{name}</div>
```

Comment:
Consider using `p` elements for these or the Material UI Typography component: https://material-ui.com/style/typography/

Reply:
Good idea. Used some `<p>` tags in 00f77c4